examples/frame_prediction/js/entities/entities.js

```diff
@@ -0,0 +1,76 @@
+/************************************************************************************/
+/*                                                                                  */
+/*     Base entity that predicts its position based on the last update              */
+/*                                                                                  */
+/************************************************************************************/
+game.FramePredictionEntity = me.Entity.extend({
+   
+    init: function(x, y, settings) {
+        this._super(me.Entity, "init", [x, y , settings]);
+    },
+    
+    draw: function(renderer) {
+        // Predict the next position to draw at based on velocity
+        var updateInterval = (me.timer.getTime() - me.timer.lastUpdate) / ( (1000/me.sys.updatesPerSecond) );
```

Comment:
the good thing is that the second paramater is actually not documented, and as I was saying earlier only used by the `me.TMXLayer` object, so we are really free to do whatever we prefer with it (adding parameters will probably break a few games though, for all the people that are redefining the draw method and calling the parent one)

Renderable API (note the typo on the argument name) :
http://melonjs.github.io/docs/me.Renderable.html#draw